### PR TITLE
OSDOCS-3589 remove etcd bullet from post-install docs

### PR DESCRIPTION
**Pull request by lahinson**
[OSDOCS-3589](https://issues.redhat.com//browse/OSDOCS-3589): Remove etcd bullet from post-installation docs 

Version(s): 4.11

Issue: https://issues.redhat.com/browse/OSDOCS-3589

Link to docs preview: https://deploy-preview-45584--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/storage-configuration.html#other-specific-application-storage-recommendations

Additional information:
This PR contains one minor change that is requested in [OSDOCS-3589](https://issues.redhat.com//browse/OSDOCS-3589). Tami Love and I are splitting up the work that is requested in that Jira. This change is minor -- the removal of one bullet in a list.

### CODE REVIEW

File: modules/recommended-configurable-storage-technology.adoc [hunk 1]
```diff
@@ -120,7 +120,6 @@ Application use cases vary from application to application, as described in the
 
 == Other specific application storage recommendations
 
-* {product-title} Internal `etcd`: For the best `etcd` reliability, the lowest consistent latency storage technology is preferable.
 * It is highly recommended that you use `etcd` with storage that handles serial writes (fsync) quickly, such as NVMe or SSD. Ceph, NFS, and spinning disks are not recommended.
```
Comment (tjungblu) [inline modules/recommended-configurable-storage-technology.adoc:123]: that bullet still mentions etcd, can we remove that one too? thanks!

**Review by tjungblu — CHANGES_REQUESTED** [review]

File: modules/recommended-configurable-storage-technology.adoc (hunk 1 above)
Comment (lahinson) [reply modules/recommended-configurable-storage-technology.adoc:123]: Will do!

**Comment by lahinson** [comment]
Hi @tjungblu - I removed the other etcd bullet as you requested. You can see the preview here: https://deploy-preview-45584--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/storage-configuration.html#other-specific-application-storage-recommendations

Thanks!

**Review by tjungblu — APPROVED** [review]

**Pull request merged**